src/overlays/actors/ovl_En_Door/z_en_door.h

```diff
@@ -4,14 +4,57 @@
 #include <ultra64.h>
 #include <global.h>
 
+/**
+ * Actor Parameters
+ * 
+ * |                  |       |             |
+ * | Transition Index | Type  | Double Door | Switch Flag OR Text Id - 0x0200
+ * |------------------|-------|-------------|---------------------------------
+ * | 0 0 0 0 0 0      | 0 0 0 | 0           | 0 0 0 0 0 0 
+ * | 6                | 3     | 1           | 6
+ * |
+ * 
+ * Transition Index     1111110000000000
+ * Type                 0000001110000000
+ * Double Door          0000000001000000
+ * Switch Flag          0000000000111111
+ * Text id - 0x0200     0000000000111111
```

Comment:
Would be nice to comment on:

"Transition Index" being set by the actor engine
(cf `Actor_SpawnTransitionActors`, it has `(i << 0xA) + transitionActor->params`)
so, those should be zeros in the scene transition actors list (relevant scene header command `func_80098C68`)

"Switch Flag" is for type DOOR_LOCKED only

"Text id - 0x0200" is for type DOOR_TALKING only